torch/testing/_internal/common_methods_invocations.py

```diff
@@ -789,10 +789,13 @@ def method_tests():
          lambda usv: (usv[0][..., :, :(S - 2)], usv[1], usv[2])),
         ('qr', (S, S), (False,), 'square_single', (), NO_ARGS, [skipCPUIfNoLapack, skipCUDAIfNoMagma]),
         ('qr', (S, S - 2), (True,), 'tall_single' , (), NO_ARGS, [skipCPUIfNoLapack, skipCUDAIfNoMagma]),
+        ('qr', (S - 2, S), (False,), 'wide_single' , (), NO_ARGS, [skipCPUIfNoLapack, skipCUDAIfNoMagma]),
```

Comment:
I think you actually need to make the 5th element, which is an empty tuple right now into `(True,)` for the gradients to be checked.
Can you try to make that change and make sure the gradcheck passes.

Also you can activate them for the square case that was not tested before :o